Apsitvarkom.UnitTests/Models/Validation/PollutedLocationValidationTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Apsitvarkom.Models;
+using FluentValidation;
+
+namespace Apsitvarkom.UnitTests.Models.Validation
+{
+    internal class PollutedLocationValidationTests
+    {
+        private static readonly IValidator<PollutedLocation> Validator = new PollutedLocationValidator();
+        private static readonly Guid dummyGuid = Guid.NewGuid();
+
+        private static readonly PollutedLocation[] ValidPollutionLocations =
+        {
+            new()
+            {
+                Id = dummyGuid,
+                Location = new Location
+                {
+                    Latitude = 35.929673,
+                    Longitude = -78.948237,
+                },
+                Notes = "I love writting tests",
+                Progress = 11,
+                Radius = 5,
+                Severity = Enumerations.LocationSeverityLevel.High,
+                Spotted = new DateTime(2002, 5, 23)
+            },
+            new()
+            {
+                Id = dummyGuid,
+                Location = new Location
+                {
+                    Latitude = -90,
+                    Longitude = 180,
+                },
+                Progress = 0,
+                Radius = 1,
+                Severity = Enumerations.LocationSeverityLevel.Low,
+                Spotted = new DateTime(1410, 7, 15)
+            },
+        };
+
+            private static readonly PollutedLocation[] InvalidPollutedLocations =
+            {
+            new()
+            {
+                // Invalid Location coordinates
+                Id = dummyGuid,
+                Location = new Location
+                {
+                    Latitude = 210,
+                    Longitude = -720,
+                },
+                Notes = "Hey Ho",
+                Progress = 0,
+                Radius = 0,
+                Severity = Enumerations.LocationSeverityLevel.High,
+                Spotted = new DateTime(1990, 3, 11)
+            },
+            new()
+            {
+                // Location (special case - not a primitive) is missing
+                Id = dummyGuid,
+                Notes = "",
+                Progress = 0,
+                Radius = 0,
+                Severity = Enumerations.LocationSeverityLevel.Low,
+                Spotted = new DateTime(2022, 9, 24)
+            },
+            new()
+            { 
+                // Invalid Progress and Radius
```

Comment:
I would split these two cases as if the progress validator was not correct our tests would still pass.

Reply:
You mean have a separate case for invalid Progress and Radius?

Reply:
Yeah, you understood that right. This way we can track not only if at least one of these properties fail, but also make sure each property is validated correctly.